cmd/minikube/cmd/start_flags.go

```diff
@@ -94,7 +94,7 @@ const (
 	interactive             = "interactive"
 	waitTimeout             = "wait-timeout"
 	nativeSSH               = "native-ssh"
-	minUsableMem            = 953  // 1GB In MiB: Kubernetes will not start with less
+	minUsableMem            = 1700 // 2GB In MiB: Kubernetes (kubeadm) will not start with less
```

Comment:
The comment is slightly misleading since 2GB is 1907MiB, maybe change 2GB to 1.78GB?

Reply:
It's all misleading, the real value is supposed to be 1700 MiB (and the old minimum was supposed to be 1024 MiB).

Historical minikube versions and even latest kubernetes are not politically correct, but say "MB" for 1024*1024 bytes.

Reply:
If you have a hardware with 2048, it is unlikely to show up as >=2000 - so might as well use something like 1900.
The main thing that we want to warn about now is trying to start on a hardware with 1024, our own default is 2200.

Reply:
If you start a VM with the fallback allocation of 2200, the system will report that as 2135 (after the kernel "tax")